package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "qubit-react",
-  "version": "1.1.2",
+  "version": "1.1.3",
```

Comment:
Will be a minor release, it's not really a bug fix, it's adding support for extra versions. But we do version bumps in separate commits anyway when we release so can remove this change from the PR